src/c++/perf_analyzer/client_backend/triton/triton_client_backend.cc

```diff
@@ -774,6 +833,19 @@ TritonInferRequestedOutput::SetSharedMemory(
   return Error::Success;
 }
 
+Error
+TritonInferRequestedOutput::SetOptions(const std::shared_ptr<Options> options)
+{
+  if (std::shared_ptr<TritonInferRequestedOutput::HttpOptions> http_options =
+          std::dynamic_pointer_cast<TritonInferRequestedOutput::HttpOptions>(
+              options)) {
+    std::shared_ptr<tc::InferRequestedOutput::HttpOptions> tc_options{
+        std::make_shared<tc::InferRequestedOutput::HttpOptions>()};
+    tc_options->json_data = http_options->json_data;
+    RETURN_IF_TRITON_ERROR(output_->SetOptions(tc_options));
```

Comment:
Why not just set the `options` directly instead of making a copy?

```
    RETURN_IF_TRITON_ERROR(output_->SetOptions(options));
```

Reply:
Irrelevant now 👍